src/PanelTraits/AutoSet.php

```diff
@@ -102,6 +103,9 @@ public function getFieldTypeFromDbColumnType($field)
             break;
 
             case 'text':
+                return 'textarea';
```

Comment:
Should this be a run-on? If it does fall through, should ALL of these be marked as such (it's PSR1, 2 or Symfony style but I forget which)?